### ISSUE TITLE
Page Discovery URL parsing fails depending on the number of slashes

### ISSUE THREAD

**Issue by gberl001** [IL2648/fuzzer#10]
When running fuzz for http://localhost:8080 I received

```
guessing for http://localhost:8080
http://index.tiff
http://index.ps
```

I assume, unless I'm wrong which is entirely possible, it is supposed to say 

```
http://localhost:8080/index.tiff
....
```

If this is the case, you can use the library I used and call the command 

```
urlParts = urlparse.urlparse(URL)
```

Then you can build your URL with the following

```
    urlParts[0] + '://' + urlParts[1] + '/' + word + extension
ex.       http + '://' + localhost:8080 + '/' + abcdefg.ext
```

**Comment by gberl001** [comment]
There is another function called split which will actually pull out even more detail (parameters, args etc)

**Comment by cburtner007** [comment]
Thanks! :) I'll keep brushing up what we have

**Comment by gberl001** [comment]
Yeah, I'm not sure how well this is going to merge... I've had to make some changes to a lot of stuff, not big, just some if blank is None : Blah kind of stuff and some try blocks.  In any case, I can help merge since I will end up causing the most merge pain... if you want.  It's probably better just to manually cherry pick the stuff when it's side by side.

**Comment by cburtner007** [comment]
Well that's not reassuring =\  Alright... I'll either pull you into an online call, or we'll have to come together in a meeting to pair-program the merge =|

**Comment by gberl001** [comment]
It's nothing to worry about, I didn't change the functionality of the code, just error checking that either quietly returns or prints that there was an error so we no longer get a 2 page document of exceptions when it fails.

I'm good for a meeting if you're up to it, maybe Monday?  We can talk later (outside of Git) but in any case, I think I'll commit my changes to a different branch and work on the new stuff without all of my error handling added in.  That way, when I submit a pull request it will only be the changes to the skeleton part... we can add error handling later since it's going to muck it all up doing it now.

**Comment by cburtner007** [comment]
I'm somewhat worried about it, because it sounds like you're doing the work that I was doing also :(

**Comment by gberl001** [comment]
I was only doing it so it would run with the site I was testing... you don't have to incorporate any of it if you don't want.  I refer back to my statement in the first meeting I had with you, I tend to do more work than I need to sometimes.  I pushed it to a separate branch so you we can do whatever, I've switched over to the original branch to work on the new submission.

Also, I thought you were only doing error handling for the args and I only knew that because you commented on the issue.  This was your last comment that I saw

```
So, I should work on the skeleton then? And then be responsible for the submission...?
```

so I apologize if I'm stepping on your toes but again, don't worry about it, you are responsible for submission so if you don't want to include it that's okay with me.  I'm mostly messing around to get to know Python and how to handle these errors.

Pretend as if I have done nothing... I will discuss nothing of it until we are done with the functionality for the next submission.  Only then will I ask if you want to compare and possibly incorporate some of the error handling.

**Comment by cburtner007** [comment]
Not a problem! Alright, let us not waste any more of your time with this discussion :) Thanks for communicating with me!
